cogs/login.py

```diff
@@ -0,0 +1,146 @@
+    @commands.command()
+    async def login(self, ctx: Context):
+        # Create Embed with Python, reads from json config file
+        embed=Embed()
+        embed.title = "Various Login Information"
+        embed.description = "Please do not share this information with anyone else!"
+        embed.set_footer(text="For Mobile Friendly, copy the password using the Selection Menu below")
+        embed.color = Color.blue()
+
+        embed.add_field(
+            name = ":orange_circle: Crunchyroll :orange_circle:",
+            value = "u: " + config["Crunchyroll"]["login"] + "\np: " + config["Crunchyroll"]["password"], 
+            inline=True
+        )
+        embed.add_field(
+            name = ":blue_circle: Disney+* :blue_circle:",
+            value = "u: " + config["Disney+"]["login"] + "\np: " + config["Disney+"]["password"], 
+            inline=True
+        )
+        embed.add_field(
+            name = ":black_circle: HBO Max :black_circle:",
+            value = "u: " + config["HBO Max"]["login"] + "\np: " + config["HBO Max"]["password"], 
+            inline=True
+        )
+        embed.add_field(
+            name = ":green_circle: Hulu* :green_circle:",
+            value = "u: " + config["Hulu"]["login"] + "\np: " + config["Hulu"]["password"], 
+            inline=True
+        )
+        embed.add_field(
+            name = ":red_circle: Netflix :red_circle:",
+            value = "u: " + config["Netflix"]["login"] + "\np: " + config["Netflix"]["password"], 
+            inline=True
+        )
+        embed.add_field(
+            name = ":football: NFL+ :football:",
+            value = "u: " + config["NFL+"]["login"] + "\np: " + config["NFL+"]["password"], 
+            inline=True
+        )
+        embed.add_field(
+            name = ":white_circle: Paramount+ :white_circle:",
+            value = "u: " + config["Paramount+"]["login"] + "\np: " + config["Paramount+"]["password"], 
+            inline=True
+        )
+        embed.add_field(
+            name = ":yellow_circle: Peacock :yellow_circle:",
+            value = "u: " + config["Peacock"]["login"] + "\np: " + config["Peacock"]["password"], 
+            inline=True
+        )
+        embed.add_field(
+            name = ":nazar_amulet: Viki (KDramas) :nazar_amulet:",
+            value = "u: " + config["Viki"]["login"] + "\np: " + config["Viki"]["password"], 
+            inline=True
+        )
+        embed.add_field(
+            name = ":brown_circle: Showtime :brown_circle:",
+            value = "u: " + config["Showtime"]["login"] + "\np: " + config["Showtime"]["password"], 
+            inline=True
+        )
+
+        select = Select(                                        # Select Method that Creates Selection Menu for Embed
+            placeholder= "Choose a streaming service...",       # Placeholder Text
+            options=[                                           # Selection Options
+                discord.SelectOption(
```

Comment:
Same here as above. Since options takes in a list, you can generate a list of SelectOption